src/SFA.DAS.Tools.AnalyseErrorQueues.Domain/ApplicationConfiguration.cs

```diff
@@ -0,0 +1,34 @@
+﻿namespace SFA.DAS.Tools.AnalyseErrorQueues.Domain.Configuration
```

Comment:
These configurations are also defined in separate files below - remove one of the definitions

Reply:
Removed unused files